sdk_container/src/third_party/coreos-overlay/sec-policy/selinux-base/selinux-base-2.20221101-r4.ebuild

```diff
@@ -132,7 +122,7 @@ src_compile() {
 
 	for i in ${POLICY_TYPES}; do
 		cd "${S}/${i}" || die
-		# flatcar changes
+		emake base
 		emake base BINDIR="${ROOT}/usr/bin" NAME=$i SHAREDIR="${ROOT%/}"/usr/share/selinux \
 			LD_LIBRARY_PATH="${ROOT}/usr/lib64:${LD_LIBRARY_PATH}" -C "${S}"/${i}
```

Comment:
I think this is just wrong. The ROOT variable can only be used in `pkg_*` functions (see https://devmanual.gentoo.org/ebuild-writing/variables/index.html).

Do you know why we have this change? Could you add an inline comment like `# Flatcar: yadda yadda`?